typescript/sdk/src/token/EvmERC20WarpRouteReader.ts

```diff
@@ -140,43 +140,34 @@ export class EvmERC20WarpRouteReader extends HyperlaneReader {
   async deriveWarpRouteVirtualConfig(
     chain: ChainName,
     address: Address,
-    virtualConfig: { contractVerificationStatus: Record<string, boolean> } = {
-      contractVerificationStatus: {},
-    },
   ): Promise<HypTokenRouterVirtualConfig> {
-    try {
-      const verificationStatus =
-        await this.contractVerifier.getContractVerificationStatus(
-          chain,
-          address,
-          this.logger,
-        );
+    const virtualConfig: HypTokenRouterVirtualConfig = {
+      contractVerificationStatus: {},
+    };
 
-      if (verificationStatus.isVerified)
-        virtualConfig.contractVerificationStatus[verificationStatus.name] =
-          true;
+    const contractType = (await isProxy(this.provider, address))
+      ? 'Proxy'
+      : 'Implementation';
```

Comment:
Do we have an enum/ constant variable with the possible values for each contract type? if not i think it would be a good addition to avoid magic strings